collective/jsonify/wrapper.py

```diff
@@ -911,9 +937,101 @@ def get_redirects(self):
             from zope.component import getUtility
             from plone.app.redirector.interfaces import IRedirectionStorage
             storage = getUtility(IRedirectionStorage)
-            redirects = storage.redirects('/'.join(self.context.getPhysicalPath()))
+            redirects = storage.redirects(
+                '/'.join(self.context.getPhysicalPath()))
             if redirects:
                 # remove site name (e.g. "/Plone") from redirect paths
-                self['_old_paths'] = [r[len(self.portal_path):] for r in redirects]
+                self['_old_paths'] = [r[len(self.portal_path):]
+                                      for r in redirects]
         except:  # noqa: E722
             pass
+
+    def get_portlets(self):
+        """ Get portlet assignments """
+
+        from zope.component import getUtility
+        from zope.component import getMultiAdapter
+        from plone.portlets.interfaces import IPortletManager
```

Comment:
I think adding imports from plone.portlets breaks support for plone versions older than 3.0